yaki_mobile/pubspec.yaml

```diff
@@ -91,6 +91,7 @@ dependencies:
 
   built_value: ^8.4.3
   json_annotation: ^4.8.0
+  flutter_spinkit: ^5.1.0
```

Comment:
is it used ? is it coherent with the design system of the app ?

Reply:
Was about to try it if status page  cascade was working for the "data refresh waiting time", but, removed since not used